### DISCUSSION TITLE
Fungerar inte att ladda ned från svtplay.se

### DISCUSSION THREAD

**Discussion by Daniel333xxx** [stefansundin/privatkopiera#89] [Q&A]
Hej Stefan!
För ett halvår sedan läste jag dina instruktioner, även den där det står att man ska skriva "cd Skrivbord" först. Jag gjorde så och det fungerade inte. Däremot fungerade det bra om jag klistrade in den kopierade texten direkt. Nu fungerar inte det längre, och det står "Permission denied". Om jag nu skriver "cd Skrivbord", klistrar in den kopierade texten efteråt och trycker på Enter, så står det bara "hittar inte sökvägen". Det spelar heller ingen roll om jag skriver "cd Desktop" först. Här kommer skärmdumpar över de olika stegen jag gjort. Först från Privatkopiera, sedan från cmd.exe:

![Privatkopiera 210518](https://user-images.githubusercontent.com/81932256/122673049-f80cf900-d1ce-11eb-8e25-34f806e42271.png)

![cmd exe 210518](https://user-images.githubusercontent.com/81932256/122673064-0824d880-d1cf-11eb-94a0-d8aa8a0a9f49.png)

Är det någon som vet vad som är fel? Hoppas kunna ladda ned igen!
/Daniel

**Comment by Daniel333xxx** [comment]
Jag har förresten Windows 10 PRO (64-bitars).

**Comment by Bub0** [comment]
> Hej Stefan!
> För ett halvår sedan läste jag dina instruktioner, även den där det står att man ska skriva "cd Skrivbord" först. Jag gjorde så och det fungerade inte. Däremot fungerade det bra om jag klistrade in den kopierade texten direkt. Nu fungerar inte det längre, och det står "Permission denied". Om jag nu skriver "cd Skrivbord", klistrar in den kopierade texten efteråt och trycker på Enter, så står det bara "hittar inte sökvägen". Det spelar heller ingen roll om jag skriver "cd Desktop" först. Här kommer skärmdumpar över de olika stegen jag gjort. Först från Privatkopiera, sedan från cmd.exe:
> 
> ![Privatkopiera 210518](https://user-images.githubusercontent.com/81932256/122673049-f80cf900-d1ce-11eb-8e25-34f806e42271.png)
> 
> ![cmd exe 210518](https://user-images.githubusercontent.com/81932256/122673064-0824d880-d1cf-11eb-94a0-d8aa8a0a9f49.png)
> 
> Är det någon som vet vad som är fel? Hoppas kunna ladda ned igen!
> /Daniel

Det ska funka med desktop, även på svensk version..
Testa cd C:\Users\ANVÄNDARE\Desktop
Ändra ANVÄNDARE till din användare. :-)

**Reply by Daniel333xxx** [reply]
Tackar ödmjukast! Ja nu fungerar det som sagt!

**Comment by stefansundin** [comment]
Hej. Ursäkta för sent svar. Som @Bub0 redan har sagt så har din kommandotolk startat i `C:\Windows\System32`, och du måste navigera till `C:\Users\ANVÄNDARE\Desktop`. Det kan vara annorlunda om du har svenskt Windows, så du kan först navigera till `C:\` med `cd C:\` och sedan lista filerna med `dir` för att hitta rätt sökväg.

Jag tror att din kommandotolk startade i `C:\Windows\System32` för att den har på något vis blivit startad med administratör behörigheter, vilken egentligen inte borde vara nödvändigt. Men det borde fungera ändå om du lyckas navigera till skrivbordet.

Hoppas du löser det.

**Reply by Daniel333xxx** [reply]
Hej!

Jag tackar ödmjukast! Om jag skriver cd C:\Users\DATA\Desktop kommer jag rätt och kan klistra in och ladda ned! (Har Windows 10 PRO, 64-bitars, om det kan ha något att göra med att jag inte kan skriva bara cd Desktop.)

/Daniel

**Comment by BacchusFLT** [comment]
Daniel,

För att kunna ladda ner måste man kunna skriva till nuvarande bibliotek. Som användare har man inte skrivrättigheter till system biblioteken, windows och dess underkataloger. Detta för att man inte skall kunna ställa till dumheter av ren okunskap. Antingen för att man gör det själv eller för att man startar ett illvilligt program med samma priviliegier som man som användare har.

Så det du vill göra är att ställa in nuvarande bibliotek till ett bibliotekt där du har skrivrättigheter men också där du lätt hittar filerna. Som jag förstår så vill du ladda ner till ditt skrivbord. Där är dom enkla att hitta och dit har du skrivrättigheter, så det är ett bra val som temporär lagring.

Du har redan en metod för att starta kommandoskalet ("cmd") och som standard pekar det på systembiblioteken. Det syns på bilderna ovan. Jag har svensk windows 10 och när jag skriver "cd Desktop" så blir skrivbordet aktuellt bibliotek. Jag vet inte varför detta inte fungerar för dig, med här är några förslag på vad du kan prova:

Alternativ 1:
Starta skalet och skriv "cd desk". Tryck inte Retur, utan tryck istället på Tab knappen. Då BORDE det bli så att det du skrivit byts ut mot "cd Desktop". Tryck nu på Retur. Då BORDE det vara så att du ser att aktuellt bibliotek är Desktop. Det står då typ "C:\User\[ditt användartnamn]\Desktop>"

Alternativ 2:
Försök starta ett skal som pekar på skrivbordet. Om du har skrivbordet framför dig så kan du trycka ner shift och samtidigt klicka på höger musknapp. Du får då upp en "Context menu" där ett av alternativen är "Öppna ett Powershell förnster här". Om du väljer detta så får du upp ett skal (inte standard skalet utan det utökade PowerShell skalet). Där borde du också kunna klistra in Privatkopiera strängen för att ladda ner. Powershell är inte 100% samma så om du väljer detta alternativ finns det så klart en risk att det inte beter sig som du kanske förväntar dig på något annat sätt.

Alternativ 3:
Öppna utforskaren (Enklast: Tryck Windows knappen + E). Navigera till Skrivbordet genom att klicka på biblioteken i vänstersidan. Placera cursorn i inmatningsfältet, marketa all text och ersätt den med "cmd" och tryck på return. 

Alternativ 2 + 3 finns beskrivna här:
https://www.youtube.com/watch?v=bgSSJQolR0E

**Reply by Daniel333xxx** [reply]
Hej Pontus!

Nu fungerar det! Tack för hjälpen! Jag skrev cd C:\Users\DATA\Desktop och kom till Desktop, kunde klistra in och ladda ned. Fick det dock inte att fungera genom att skriva cd desk och trycka på Tab-knappen eller skriva bara cd Desktop och Retur.  (Har Windows 10 PRO, 64-bitars, om det kan ha något att göra med att jag inte kan skriva bara cd Desktop.)

/Daniel
